src/Microsoft.PowerShell.Commands.Utility/commands/utility/Measure-Object.cs

```diff
@@ -708,8 +739,18 @@ private void AnalyzeString(string strValue, Statistics stat)
         /// </summary>
         private void AnalyzeNumber(double numValue, Statistics stat)
         {
-            if (_measureSum || _measureAverage)
+            if (_measureSum || _measureAverage || _measureStdDeviation)
+            {
+                stat.sumPrevious = stat.sum;
                 stat.sum += numValue;
+            }
+            if (_measureStdDeviation && stat.count > 1)
+            {
+                double countMinusOne = stat.count - 1;
```

Comment:
I don't think that introducing the `countMinusOne` variable makes the code cleaner. Let's just use `count - 1` inline instead. To force the `(count - 2) / (count -
 1)` to be a double division, instead of integer division, simply cast one of them to double, i.e.

```
(count - 2) / (count - 1.0)
```

Reply:
+1  I believe it is for compiler optimization.